breadbox/breadbox/schemas/dataset.py

```diff
@@ -119,6 +119,13 @@ class MatrixDatasetParams(SharedDatasetParams):
         ),
     ] = None
 
+    data_file_format: Annotated[
+        str,
```

Comment:
Sounds like the value will be either "csv" or "parquet" only. How about we don't type it as `str` which could be any string and instead `Union[Literal["csv"], Literal["parquet"]]`?

Reply:
Good point. I'll fix.